models/articles.model.js

```
@@ -30,8 +24,20 @@ exports.fetchArticles = () => {
         return Promise.reject({ status: 404, msg: "No articles found" });
       }
       return rows;
-    })
-    .catch((err) => {
-      return Promise.reject(err);
+    });
+};
+
+exports.fetchCommentsByArticleId = (id) => {
+  return db
+    .query(
+      `SELECT comment_id, votes, created_at, author, body, article_id FROM comments WHERE article_id = $1 ORDER BY created_at DESC
+      `,
+      [id]
+    )
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "Not found" });
+      }
```

Do I want to throw an error if there are no comments? What if this is a newly posted article, when I would expect to have no comments?